app/main.py

```diff
@@ -1 +1,110 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    # Check if input command format is correct
+    if not is_input_data_correct(command):
+        return
+
+    # Get correct paths for source and destination files
+    origin_input_path, dest_input_path = command.split()[1:]
+    dest_short_path, dest_file_name = os.path.split(dest_input_path)
+    origin_short_path, origin_file_name = os.path.split(origin_input_path)
+    if not dest_file_name:
+        dest_full_path = os.path.join(dest_short_path, origin_file_name)
+    else:
+        dest_full_path = dest_input_path
```

Comment:
```suggestion
        dest_input_path = os.path.join(dest_short_path, origin_file_name)
```
dest_full_path is redundant variable name

Reply:
Ok